features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -333,3 +333,23 @@
   step %{I open signups for "Awesome Gift Exchange"}
   step %{everyone has signed up for the gift exchange "Awesome Gift Exchange"}
 end
+
+### Deleting things
+# use the same names as "everyone is signed up"
+
+Then /^no one should be signed up for "([^\"]*)"$/ do |challenge|
+  collection = Collection.find_by_title(challenge)
+  %w(myname1 myname2 myname3 myname4).each do |name|
```

Comment:
How about if I change it to `everyone's assignments for "([^\"]*)" should be deleted` and `everyone's sign ups for "([^\"]*)" should be deleted`? I don't want to rename the initial (and equally misleading) `everyone has signed up for the gift exchange "([^\"]*)"` step, but I also don't want to check more than we need to.